doc/user_guide/user_guide.md

```diff
@@ -299,7 +299,7 @@ Unless stated otherwise in the comments column, the option is required for eithe
 | bucketfs-port                |   [x]   |      |                                                 |
 | bucketfs-user                |   [x]   |      |                                                 |
 | bucketfs-password            |   [x]   |      | Env. [BUCKETFS_PASSWORD]                        |
-| bucketfs-use-https           |   [x]   |      | Optional boolean, defaults to False             |
+| bucketfs-use-https           |   [x]   |      | Optional boolean, defaults to True              |
```

Comment:
this also needs to be changed in the code

Reply:
Yes, of course. I changed it in the other PR, which is currently in review.

Reply:
Ah ok